data-otservbr-global/scripts/talkactions/player/server_info.lua

```diff
@@ -1,25 +1,39 @@
-	.. "\nMagic rate: " .. getRateFromTable(magicLevelStages, player:getBaseMagicLevel(), configManager.getNumber(configKeys.RATE_MAGIC))
-	.. "\nLoot rate: " .. configManager.getNumber(configKeys.RATE_LOOT)
-
-	local houseBuyLevel = configManager.getNumber(configKeys.HOUSE_BUY_LEVEL)
-	if (houseBuyLevel ~= nil and player:getLevel() < houseBuyLevel) then
-		text = text .. '\nLevel to buy house: ' .. houseBuyLevel
-	end
-
-	player:sendTextMessage(MESSAGE_EVENT_ADVANCE, text)
+	local configRateSkill =  configManager.getNumber(configKeys.RATE_SKILL)
+	local text = "Server Info Rates: \n"
+	.. "\nExp rate: " .. configManager.getNumber(configKeys.RATE_EXPERIENCE)
+	.. "\nSword Skill rate: " .. getRateFromTable(skills, player:getSkillLevel(SKILL_SWORD), configRateSkill) .. "x"
+	.. "\nClub Skill rate: " .. getRateFromTable(skills, player:getSkillLevel(SKILL_CLUB), configRateSkill) .. "x"
+	.. "\nAxe Skill rate: "  .. getRateFromTable(skills, player:getSkillLevel(SKILL_AXE), configRateSkill) .. "x"
+	.. "\nDistance Skill rate: " .. getRateFromTable(skills, player:getSkillLevel(SKILL_DISTANCE), configRateSkill) .. "x"
+	.. "\nShield Skill rate: " .. getRateFromTable(skills, player:getSkillLevel(SKILL_SHIELD), configRateSkill) .. "x"
+	.. "\nFist Skill rate: " .. getRateFromTable(skills, player:getSkillLevel(SKILL_FIST), configRateSkill) .. "x"
+	.. "\nMagic rate: " .. getRateFromTable(magicLevel, player:getBaseMagicLevel(), configManager.getNumber(configKeys.RATE_MAGIC)) .. "x"
+	.. "\nLoot rate: "  .. configManager.getNumber(configKeys.RATE_LOOT) .. "x"
+	.. "\nSpawns rate: " .. configManager.getNumber(configKeys.RATE_SPAWN) .. "x"
+	text = text .. "\n"
+	text = text .. "\n\nServer Info Stages Rates: \n"
+	.. "\nExp rate stages: " ..  getRateFromTable(experienceStages, player:getLevel(), expstagesrate) .. "x"
+	.. "\nSword Skill Stages rate: " .. getRateFromTable(skillsStages, player:getSkillLevel(SKILL_SWORD), configRateSkill) .. "x"
+	.. "\nClub Skill Stages rate: " .. getRateFromTable(skillsStages, player:getSkillLevel(SKILL_CLUB), configRateSkill) .. "x"
+	.. "\nAxe Skill Stages rate: " .. getRateFromTable(skillsStages, player:getSkillLevel(SKILL_AXE), configRateSkill) .. "x"
+	.. "\nDistance Skill Stages rate: " .. getRateFromTable(skillsStages, player:getSkillLevel(SKILL_DISTANCE), configRateSkill) .. "x"
+	.. "\nShield Skill Stages rate: " .. getRateFromTable(skillsStages, player:getSkillLevel(SKILL_SHIELD), configRateSkill) .. "x"
+	.. "\nFist Skill Stages rate: " .. getRateFromTable(skillsStages, player:getSkillLevel(SKILL_FIST), configRateSkill) .. "x"
+	.. "\nMagic rate: " .. getRateFromTable(magicLevelStages, player:getBaseMagicLevel(), configManager.getNumber(configKeys.RATE_MAGIC)) .. "x"
+	.. "\nLoot rate: " .. configManager.getNumber(configKeys.RATE_LOOT) .. "x"
+	.. "\nSpawns rate: " .. configManager.getNumber(configKeys.RATE_SPAWN) .. "x"
+	text = text .. "\n"
```

Comment:
you can remove this. and concatenate with the line below.
ex: remove text = text .. "\n" usage.. as the line above with concatenation.

Reply:
it works, it will be modified, thanks for the suggestion.